CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 * **Removed**
 
 * **Bug Fix**
+   * FIXED: Fix compilation errors when boost < 1.68 and libprotobuf < 3.6  [#2878](https://github.com/valhalla/valhalla/pull/2878)
```

Comment:
can you move this to the bottom of the list, we've been following that pattern. basically inside of a release the oldest stuff is at the top and the youngest stuff is at the bottom. i can see why that is somewhat unappealing though.... maybe we can change it in the next release and follow that from now on